staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -149,11 +150,22 @@ func newAdmissionMetrics() *AdmissionMetrics {
 		},
 		[]string{"name", "type", "operation", "error_type", "rejection_code", "namespace"})
 
+	webhookRequest := metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Namespace:      namespace,
+			Subsystem:      subsystem,
+			Name:           "webhook_request_total",
+			Help:           "Admission webhook request total, identified by name and broken out for each admission type (validating or admit) and operation. Additional labels specify an HTTP status code. Codes greater than 600 are truncated to 600, to keep the metrics cardinality bounded.",
+			StabilityLevel: metrics.ALPHA,
+		},
+		[]string{"name", "type", "operation", "code", "rejected", "namespace"})
```

Comment:
What if we switched rejected for permitted? It's easier for humans to reason about things stated in the positive?

Reply:
sgtm, the only drawback is the latency metric uses rejected.

Reply:
Oh, I'm sorry, I looked for other metrics and didn't see that one while skimming, only the rejection one, and we should probably be consistent, sigh.

Uh, but now that you point it out, the only thing this counter metric is adding is the response code. Why do we need an entirely new metric just for that? You should be able to get a count out of a histogram, no?

Reply:
Okay, I'll revert back to using rejected.

There is a comment thread above this discussing whether the code should be added to the latency metric or as an entirely new metric. The downside of adding this to the latency metric is it would make the cardinality explode for the histogram.